packages/core/phase2/lib/areAnyPncResults2007.test.ts

```diff
@@ -1,4 +1,4 @@
-import type { AnnotatedHearingOutcome, Offence } from "../../../types/AnnotatedHearingOutcome"
+import type { AnnotatedHearingOutcome, Offence } from "../../types/AnnotatedHearingOutcome"
 import areAnyPncResults2007 from "./areAnyPncResults2007"
```

Comment:
Is there a reason this can't be a more general function? `pncRecordHasDisposalType(2007)`

Reply:
Good point.

Reply:
It looks like a disposal type with 2007 is something meaningful and the only thing we're interested in although I see your point. There is also `areAllPncResults2007` which could be made generic. I'll refactor this.

Reply:
Have opened https://github.com/ministryofjustice/bichard7-next-core/pull/1055 to address this feedback, thanks!